lib/amp/middleware.js

```diff
@@ -73,7 +73,7 @@ const createCustomHead = originalHead => function customHead() {
    */
   if (this.$isAMP) {
     ensureKey(head, 'htmlAttrs', {})
-    head.htmlAttrs['⚡'] = undefined
+    head.htmlAttrs.amp = true
```

Comment:
Do you have any particular reason for this change? (personally, I prefer `⚡` over `amp`)

Also `undefined` is do a thing for us, it remove value of attribute (`<html amp ...>`)
Using `true` generates this HTML (`<html amp="true" ...>`)

Reply:
I try to use ⚡ but it looks like not support in new nuxt version. I try to clean and `npm install` this is what I ve found.
- ⚡ is not display in attr I don't know why
- set amp = null is not show `<html amp ...>` and amp attr is disappear.
- set amp = true is show `<html amp ...>` not `<html amp="true" ...>`

nuxt.js version is v2.9.0-26056810.eb060bed

Reply:
Based on current PR this is the HTML tag `<html data-n-head-ssr amp data-n-head="amp">` and when I change back from `head.htmlAttrs.amp = true` to `head.htmlAttrs['⚡'] = undefined` Lightning is gone like this `<html data-n-head-ssr data-n-head="">`

Reply:
another reason why I've changed ⚡ to amp is currently amp.dev example is already changed from ⚡ to amp too. 

[link](https://amp.dev/documentation/guides-and-tutorials/start/create/basic_markup/?format=websites)

Reply:
Sorry i forgot to mention it for this module. Latest version of nuxt (edge) and upcoming 2.9 is using vue-meta@2 and this is a breaking change requirement. /cc @pimlie

Emoji seems not officially recommend. This change looks good to me.